pages/proposal/[id].js

```diff
@@ -68,6 +73,9 @@ const Proposal = ({ id, defaultProposalData }) => {
     }
   }
 
+  console.log("receipt, ",receipt)
+  console.log("data",data)
+
```

Comment:
do we want take these out?

Reply:
Removed it. 

Did you manage to solve the proposal update status/vote count issue?

Reply:
Not yet, but I created a card to make sure we don't forget. https://trello.com/c/TccYv91B/69-fix-idjs-page-not-updating-immediately-after-vote

But it's okay, it can be managed in another PR.